pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -221,6 +221,51 @@ type ClusterSpec struct {
 	// +optional
 	// +nullable
 	LogCollector LogCollectorSpec `json:"logCollector,omitempty"`
+
+	// CSI Driver Options applied per cluster.
+	// +optional
+	// +nullable
+	CsiDriver CSIDriverOptionsSpec `json:"csiDriver,omitempty"`
```

Comment:
As it is here, this spec item is not nullable because it is not a pointer to a struct.

Should this be a pointer to struct (`*CSIDriverOptionsSpec`), or should `// +nullable` be removed?

Reply:
> As it is here, this spec item is not nullable because it is not a pointer to a struct.
> 
> Should this be a pointer to struct (`*CSIDriverOptionsSpec`), or should `// +nullable` be removed?

:+1: 
`nullable` was added by mistake, removed it.